### ISSUE TITLE
All hosts reporting unreachable in OMD 5.41~2024-10-06-labs-edition

### ISSUE THREAD

**Issue by systemdlete** [sni/Thruk#1403]
**Describe the bug**
After upgrading from 5.30 to 5.41 due to a bug that prevented thruk from operating, I note that every monitored host is reporting it is unreachable (even though there are no errors or warnings from service checks on any of the hosts).

**Thruk Version**
[Thruk 3.19.20241006](http://www.thruk.org/)
[OMD 5.41~2024-10-06-labs-edition](https://labs.consol.de/omd/)

**To Reproduce**
1. Upgrade to 5.41
2. Observe that all hosts are reporting unreachable.
3. Observe that service checks are reaching the hosts (other than intermittent issues due to actual service check issues)

**Expected behavior**
Behavior should continue as previously, with only those hosts that are ACTUALLY unreachable being marked so.

**Screenshots**
Prefer not to include for privacy reasons; however, can supply one if deemed worthwhile.

**Desktop (please complete the following information):**
- Devuan Daedalus (bookworm), with all latest patches applied.
- XFCE4.18
 - Browser Firefox 128.3.1esr (64bit)

**Additional context**
N/A

**Comment by sni** [comment]
what's the parents status of those hosts?

**Comment by systemdlete** [comment]
The error message is "check_icmp: Failed to obtain ICMP socket: Operation not permitted" 

Same error when I run check_icmp manually.

The status of every parent is unreachable.

**Comment by sni** [comment]
On debian, we use setcap to set the required capabilities:

```
getcap /omd/sites/*/lib/monitoring-plugins/check_icmp
/omd/sites/.../lib/monitoring-plugins/check_icmp cap_net_raw=ep
```

**Comment by systemdlete** [comment]
Did you mean "setcap" not getcap?  And if setcap needs to be run, did it fail somehow during installation of OMD 5.41?

I've used setcap for restic, for similar reason.  Using setcap does seem to make this work.

**Comment by sni** [comment]
well, setcap is used during building the packages, no manual run of setcap is required. getcap was used to show the capabilities and how they should look like.

**Comment by systemdlete** [comment]
I did not do anything with permissions on the installed image of 5.41, honest!   But somehow, the executable was left with insufficient privileges after the install completed.  And as far as I can recall the installation, there were no abnormalities or errors.

Perhaps it was the original condition that led to the upgrade to 5.41 in the first place, where omd was having errors and you suggested the upgrade, that might be the bit that caused the install to not complete properly.  (I would agree that this does not seem likely, but something must have led to the installation missing the privilege part.)  Again, I did not do anything to permissions on that or any other binary or script in the /opt/omd/versions area, nor to the files in the existing site.  Also, my config was pointing to the copy in the /opt/omd/versions binary directory, not a local copy within my site.
